gui/packages/ubuntupro/lib/pages/subscribe_now/subscribe_now_widgets.dart

```diff
@@ -61,58 +61,97 @@ class _ProTokenInputFieldState extends State<ProTokenInputField> {
     final lang = AppLocalizations.of(context);
     final theme = Theme.of(context);
 
-    return YaruExpandable(
-      header: Text(
-        lang.tokenInputTitle,
-        style:
-            theme.textTheme.bodyMedium!.copyWith(fontWeight: FontWeight.w100),
-      ),
-      expandIcon: Icon(
-        ProTokenInputField.expandIcon,
-        color: theme.textTheme.bodyMedium!.color,
-      ),
-      isExpanded: widget.isExpanded,
-      child: ValueListenableBuilder(
-        valueListenable: _token,
-        builder: (context, _, __) => Row(
-          crossAxisAlignment: CrossAxisAlignment.start,
-          children: [
-            Expanded(
-              child: TextField(
-                inputFormatters: [
-                  // This ignores all sorts of (Unicode) whitespaces (not only at the ends).
-                  FilteringTextInputFormatter.deny(RegExp(r'\s')),
-                ],
-                autofocus: false,
-                controller: _controller,
-                decoration: InputDecoration(
-                  hintText: lang.tokenInputHint,
-                  errorText: _token.errorOrNull?.localize(lang),
-                  counterText: '',
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Text(
+          lang.tokenInputTitle,
+          style:
+              theme.textTheme.bodyMedium!.copyWith(fontWeight: FontWeight.w100),
+        ),
+        const SizedBox(
+          height: 8,
+        ),
+        ValueListenableBuilder(
+          valueListenable: _token,
+          builder: (context, _, __) => Row(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              Expanded(
+                child: TextField(
+                  inputFormatters: [
+                    // This ignores all sorts of (Unicode) whitespaces (not only at the ends).
+                    FilteringTextInputFormatter.deny(RegExp(r'\s')),
+                  ],
+                  autofocus: false,
+                  controller: _controller,
+                  decoration: InputDecoration(
+                    hintText: lang.tokenInputHint,
+                    error: _token.errorOrNull?.localize(lang) != null
+                        ? Padding(
+                            padding: const EdgeInsets.only(top: 4),
+                            child: Row(
+                              children: [
+                                Icon(
+                                  Icons.cancel,
+                                  color: YaruColors.of(context).error,
```

Comment:
Is there a preference for `theme.colorScheme` vs `YaruColors.of`?

Reply:
For the error case they would mean the same, but `YaruColors` offers `.link`, `.warning` and `.success` which I don't think `theme.colorScheme` does. So, YaruColors is preferrable when those semantics may change. For ease, let's just stick to `YaruColors`.